posts/inside-rust/2020-06-23-Ownership-Std-Implementation.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: "Ownership of the standard library implementation"
+author: Ashley Mannix
+team: The Libs team <https://www.rust-lang.org/governance/teams/library>
+---
+
+Our Rust project is a large and diverse one. Its activities are broadly coordinated by teams that give the community space to find and contribute to the things that matter to them. We’re trialing a reorganization of standard library activities between the Libs and Compiler teams. Going forward, the Libs team will own just the public API of the standard library, and the Compiler team will own its implementation. The goal of this separation of concerns is to better suit the interests of both teams to better support the needs of the standard library. Why do we think this is necessary?
```

Comment:
It feels like this blog post doesn't say much about this being a "trial" -- to be honest, I suspect we could as well just **commit** to the new setup, but it might be worth adding a section like 

### Trial Period

For now, we are attempting this on a trial basis. We expect to re-evaluate the setup later this year and decide whether to make it permanent.